com.ibm.wala.ide.jdt.test/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -89,7 +89,7 @@ org.eclipse.jdt.core.compiler.problem.reportMethodCanBeStatic=ignore
 org.eclipse.jdt.core.compiler.problem.specialParameterHidingField=disabled
 org.eclipse.jdt.core.compiler.problem.staticAccessReceiver=ignore
 org.eclipse.jdt.core.compiler.problem.suppressOptionalErrors=enabled
-org.eclipse.jdt.core.compiler.problem.suppressWarnings=enabled
+org.eclipse.jdt.core.compiler.problem.suppressWarnings=disabled
```

Comment:
For this one ECJ was complaining that a `@SuppressWarnings` was unnecessary, but it was necessary for the Error Prone checker.  This is for code compiled by Maven only so I'm not concerned about dialing down this check (which is only for unnecessary suppressions anyway)